plugins/chat/spec/lib/chat_sdk/message_spec.rb

```diff
@@ -168,24 +166,22 @@
   end
 
   describe ".stream" do
-    fab!(:message_1) { Fabricate(:chat_message, message: "first") }
+    fab!(:message_1) { Fabricate(:chat_message, message: "first\n") }
     before { message_1.update!(streaming: true) }
 
     it "streams" do
```

Comment:
Maybe we need some reference to strip_white_spaces here as they are allowed for streaming

Reply:
No because here we are testing update, but you are right that this is confusing and I actually don't correctly test create with stream in the sdk. I just pushed a commit which should have it correctly :+1: